src/styles/scss/tools/variables.spacings.scss

```diff
@@ -0,0 +1,40 @@
+$s-xs: 4px;
+$s-sm: 8px;
+$s-md: 16px;
+$s-lg: 24px;
+$s-xl: 32px;
+$s-2xl: 44px;
+$s-3xl: 60px;
+$s-4xl: 80px;
+$s-5xl: 100px;
+$s-6xl: 112px;
+$s-7xl: 215px;
+
+// Odd ones out, that might need to be cleaned.
+$s-odd-2: 2px;
+$s-odd-20: 20px;
+$s-odd-22: 22px;
+$s-odd-34: 34px;
+$s-odd-35: 35px;
+$s-odd-40: 40px;
+$s-odd-45: 45px;
+$s-odd-48: 48px;
+$s-odd-50: 50px;
+$s-odd-56: 56px;
+$s-odd-64: 64px;
+$s-odd-96: 96px;
+$s-odd-157: 157px;
```

Comment:
Som du også selv skriver, "might need to be cleaned". For mig ligner det en struktur vi ikke burde bruge fremover, så derfor tænker jeg;

Siden de bliver brugt i "legacy_spacings" variablen, kan vi så ikke rykke dem derud til at starte med. Så kan vi/andre evt. refaktorere componenter som bruger de størrelser til enten at bruge legacy_spacings, eller helt smide dem ud og lave om ved at ændre de komponenter der bruge dem hvis det er bedre. 

Det som eks. "$s-odd-157" bliver brugt til synes jeg er underligt, eller måske overser jeg noget. 

Hvis det er at styre at "advanced search" er centreret, tænker jeg vi burde arbejdet med "max-width" variablerne i `variables.layout.scss` eller lignende afhængigt af hvad vi bruger.

Reply:
Agreed. I'll rollback all the $s-odd- variables.